ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/ReportReviewForm.tsx

```diff
@@ -66,7 +68,9 @@ function ReportReviewForm({ formValues }: ReportReviewFormParams): ReactElement
     const deliveryDestinations =
         formValues.deliveryDestinations.length !== 0 ? (
             formValues.deliveryDestinations.map((deliveryDestination) => (
-                <li key={deliveryDestination.notifier?.id}>{deliveryDestination.notifier?.name}</li>
+                <li key={deliveryDestination?.notifier?.id}>
```

Comment:
Nit: `formValues.deliveryDestinations` is defined as `DeliveryDestination[]` - there shouldn't be any need for the optional chaining throughout this file until you get to `deliveryDestination.notifier?`.